os/services/lwm2m/lwm2m-engine.h

```diff
@@ -114,6 +116,14 @@ void lwm2m_notify_object_observers(lwm2m_object_instance_t *obj,
 
 void lwm2m_engine_set_opaque_callback(lwm2m_context_t *ctx, lwm2m_write_opaque_callback cb);
 
+#if LWM2M_Q_MODE_ENABLED
```

Comment:
Ditto here.

I think those prototypes should be present here all the time, I cannot see a reason why not.

Reply:
The same, we need an agreement.